File: DevExpress-Demo/CS/DialogsDemo.Wpf/Modules/TabControl.WebBrowser/MainViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;

namespace CommonDemo.TabControl.WebBrowser {
    public class MainViewModel {
        public void LaunchDemo() {
            this.GetService<IWindowService>().Show("WebBrowserView", null, this);
        }
    }
}

File: DevExpress-Demo/VB/DialogsDemo.Wpf/Modules/TabControl.WebBrowser/MainViewModel.vb
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.POCO

Namespace CommonDemo.TabControl.WebBrowser

    Public Class MainViewModel

        Public Sub LaunchDemo()
            GetService(Of IWindowService).Show("WebBrowserView", Nothing, Me)
        End Sub
    End Class
End Namespace
